app/models/response.rb

```diff
@@ -4,4 +4,16 @@ class Response < ActiveRecord::Base
 
   validates :poll_id, presence: true
   validates :choice_id, presence: true
+
+  def question
+    self.choice.question.text
+  end
+
+  def text
+    self.choice.text
+  end
+
+  def question_choices
+    self.choice.question.choices
+  end
```

Comment:
@philsof I was probably wrong for the application that you need.  Based on what you're saying, maybe it would be better to modify our associations in the response model.  If we add "has_one :question, through: :choice" in the Response class, then you would have the question for a response at response.question.  The choices would be available at response.question.choices, or we could shorten that with a method (response.choices, maybe).